server/routes/createTodos.js

```diff
@@ -0,0 +1,9 @@
+const { connect } = require('hyper-connect')
+require('dotenv').config()
```

Comment:
I notice you're loading `dotenv` multiple times. Typically, this is only done once, when the app starts. It's common to load a `.env` only when in development mode ie. `NODE_ENV === 'development'`. In upper environments, environment variables are loaded differently. `dotenv` is a convenient way to load environment from a `.env` file during development.

I would consider moving this to the top of your `server/index.js` file and then removing it in all of these route files.

Reply:
Will do. I was following the example in the data-connect workshop. I will definitely make those changes.